recipes/natural_language_processing/chatbot/bootc/Containerfile

```diff
@@ -24,19 +24,32 @@ ARG APPIMAGE=quay.io/ai-lab/${RECIPE}:latest
 ARG SERVERIMAGE=quay.io/ai-lab/llamacpp-python:latest
 
 # Add quadlet files to setup system to automatically run AI application on boot
-COPY quadlet/${RECIPE}.kube quadlet/${RECIPE}.yaml quadlet/${RECIPE}.image /usr/share/containers/systemd
+COPY quadlet/${RECIPE}.kube quadlet/${RECIPE}.yaml /usr/share/containers/systemd
 
 # Modify quadlet files to match the server, model and app image
 RUN sed -e "s|SERVERIMAGE|${SERVERIMAGE}|" \
     -e "s|APPIMAGE|${APPIMAGE}|g" \
     -e "s|MODELIMAGE|${MODELIMAGE}|g" \
     -i \
-    /usr/share/containers/systemd/${RECIPE}.image \
     /usr/share/containers/systemd/${RECIPE}.yaml
 
+# Because images are prepulled, no need for .image quadlet
+# COPY quadlet/${RECIPE}.image /usr/share/containers/systemd
+# RUN sed -e "s|SERVERIMAGE|${SERVERIMAGE}|" \
+#    -e "s|APPIMAGE|${APPIMAGE}|g" \
+#    -e "s|MODELIMAGE|${MODELIMAGE}|g" \
+#    -i \
+#    /usr/share/containers/systemd/${RECIPE}.image
+
+# Setup /usr/lib/containers/storage as an additional store for images.
+# Remove once the base images have this set by default.
 RUN sed -i -e '/additionalimage.*/a "/usr/lib/containers/storage",' \
         /etc/containers/storage.conf
 
+# Prepull the model, model_server and application to prepopulat the system.
 RUN podman pull --root /usr/lib/containers/storage ${SERVERIMAGE}
 RUN podman pull --root /usr/lib/containers/storage ${APPIMAGE}
 RUN podman pull --root /usr/lib/containers/storage ${MODELIMAGE}
+
+# Added for running as an OCI Container to prevent Overlay on Overlay issues.
```

Comment:
Maybe let's link to https://github.com/CentOS/centos-bootc/issues/282 because there was some good discussion there about this; it's a complex topic.  Longer term as we go down the composefs path, maybe we can go to using cfs-fuse for the container-in-container case?